hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,19 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](./) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
```

Comment:
Could you change the `documentation` link-text to `bootmodes documentation` please?

@JamesH65 Should we prefer a link-url like `./README.md` or is `./` okay?

Reply:
Ah!
I tried `./readme.md` but that didn't work `./README.md` does work. I thought URL's weren't case sensitive; but ofc it is case sensitive when referring to a file.

I strongly agree with you on `./README.md` as it's more direct/obvious to where it leads.
In fact; I've changed them to `README.md` which better matches the way these references are/were done in the rest of the document(s).

Reply:
>I thought URL's weren't case sensitive; but ofc it is case sensitive when referring to a file.

Generally URLs are case sensitive. The exception is Windows servers, where they are not case sensitive.